extend-vnet-to-multi-vnet/azuredeploy.json

```diff
@@ -0,0 +1,286 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+
+    "parameters": {
+
+        "virtualNetworkName1": {
+            "type": "string",
+            "defaultValue": "vnet01",
+            "metadata": {
+                "description": "Name of the existing first VNET. This VNET must already have a subnet named GatewaySubnet defined with a minimum /29 address prefix"
+            }
+        },
+
+        "location1": {
+            "type": "string",
+            "defaultValue": "East US",
+            "allowedValues": [
+                "West US",
+                "East US",
+                "East US 2",
+                "Central US",
+                "South Central US",
+                "North Central US",
+                "North Europe",
+                "West Europe",
+                "East Asia",
+                "Southeast Asia",
+                "Japan East",
+                "Japan West"
+           ],
+           "metadata": {
+                "description": "Region in which the first existing VNET is provisioned"
+           }
+        },
+
+        "virtualNetworkName2": {
+            "type": "string",
+            "defaultValue": "vnet02",
+            "metadata": {
+                "description": "Name of the new second VNET."
+            }
+        },
+
+        "addressPrefix2": {
+            "type": "string",
+            "defaultValue": "10.2.0.0/16",
+            "metadata": {
+                "description": "Address space for the second VNET. This address space must not overlap with the first VNET"
+            }
+        },
+
+        "subnet1Name2": {
```

Comment:
Instead of 1 and 2 it should be existing/old and new which would give better indications of these parameters to the user